### PR TITLE
meraki_device - Support for creating and modifying device notes

### LINKED ISSUE #50962: meraki_device should allow updating a device's notes
##### SUMMARY
Meraki added support for adding notes to a device. Ansible should support that.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
meraki_device

### PR DESCRIPTION
**Pull request by kbreit**
##### SUMMARY
Meraki added support for adding device notes to a device via API. This PR adds a `note` parameter which allows assigning notes to a device.

Fixes #50962

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
meraki_device

### CODE REVIEW

File: changelogs/fragments/49037-digital_ocean_tag_facts_keyerror.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+- "digital_ocean_tag_facts - Fixed KeyError exception when using tag_name parameter"
```
Comment (dagwieers) [inline changelogs/fragments/49037-digital_ocean_tag_facts_keyerror.yaml]: This is Meraki, not Digital Ocean ?

**Review by dagwieers — CHANGES_REQUESTED** [review]
LGTM, except the changelog fragment

File: changelogs/fragments/49037-digital_ocean_tag_facts_keyerror.yaml (hunk 1 above)
Comment (kbreit) [reply changelogs/fragments/49037-digital_ocean_tag_facts_keyerror.yaml]: Fixed

**Comment by kbreit** [comment]
@dagwieers Please take a look at this one.

**Review by dagwieers — APPROVED** [review]
LGTM

**Pull request merged**